libraries/Microsoft.Bot.Builder.AI.QnA/QnADialogResponseOptions.cs

```diff
@@ -41,5 +41,13 @@ public class QnADialogResponseOptions
         /// Get or set for Card no match response.
         /// </value>
         public Activity CardNoMatchResponse { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether DisplayPreciseAnswerOnly.
+        /// </summary>
+        /// <value>
+        /// Get or set whether to display Precise Answer Only or Both.
```

Comment:
>Get or set whether to display Precise Answer Only or Both. [](http://example.com/codeflow?start=12&length=58)

Indicate here what is the meaning of 'true' or 'false', and what is the default.